client/src/components/UI/__tests__/navBar.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import NavBar from '../NavBar';
+
+describe('Tests for <NavBar />', () => {
+  it('should render', () => {
+    const wrapper = shallow(<NavBar />);
+    expect(wrapper.dive().find('div').length).toBe(1);
+  });
+  it('should contain LogIn button by default', () => {
+    const wrapper = shallow(<NavBar />);
+    expect(wrapper.dive().contains('login')).toBe(true);
+  });
+  it('should not contain Leader Board by default', () => {
+    const wrapper = shallow(<NavBar />);
+    expect(wrapper.dive().contains('Leader Board')).toBe(false);
+  });
+  it('should not contain LogIn button if user has already logged in', () => {
+    const wrapper = shallow(<NavBar isLoggedIn />);
+    expect(wrapper.dive().contains('login')).toBe(false);
+  });
+  it('should contain Leader Board by default', () => {
+    const wrapper = shallow(<NavBar isLoggedIn />);
+    expect(wrapper.dive().contains('Leader Board')).toBe(true);
+  });
+});
```

Comment:
The way we decided the login and logout, none of the button will be on the navbar, It will be on the modal, user can't be allowed to login or logout in between the game, remember

Reply:
These tests are for the current functionality. When you change it you will have to update the test cases as well.

Reply:
There is no current functionality, Navbar never had login/logout button in the past

Reply:
It had and I just wrote some relevant test cases for Navbar. I did not create the Navbar so I don't know what is the status. The one who created it will update the test cases.